src/expressions.rst

```diff
@@ -624,11 +629,6 @@ it resolved to.
 :dp:`fls_5ifai8nkp5ek`
 The :t:`value` of a :t:`path expression` is the :t:`entity` that it resolved to.
 
-.. rubric:: Dynamic Semantics
-
-:dp:`fls_ed9w4jwx059`
-The :t:`evaluation` of a :t:`path expression` has no effect.
-
```

Comment:
This is not true, the evaluation of this may move out of/copy the referenced thing, this is already specified in the place expression rules though.

Reply:
Gotcha.